src/lib/components/common/ui/floatbar.ts

```diff
@@ -0,0 +1,103 @@
+import {html, css} from 'lit';
+import {property} from 'lit/decorators.js';
+import {FloatElement} from '../../custom';
+import {CustomElement} from '../../injectors';
+import './tooltip';
+
+@CustomElement()
+export class FloatBar extends FloatElement {
```

Comment:
Thanks for updating this to extend from FloatElement, forgot to catch that earlier.